.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ _Put an `x` in the boxes that apply_
 
 _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
 
-- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
+- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
```

Comment:
[Test2](../../blob/master/CONTRIBUTING.md)
`[Test2](../../blob/master/CONTRIBUTING.md)`, IMHO this is ugly and more likely to break